packages/ag-charts-community/src/sparklines-util.ts

```diff
@@ -1,26 +1,4 @@
-export * from './util/angle';
-export * from './util/attributeUtil';
-export * from './util/bboxinterface';
-export * from './util/destroy';
-export * from './util/distance';
-export * from './util/id';
-export * from './util/interpolate';
-export * from './util/json';
-export * from './util/number';
-export * from './util/padding';
-export * from './util/sanitize';
-export * from './util/value';
-
-export { isValidDate as isDate, isFiniteNumber as isNumber, isString } from './util/type-guards';
-export { extent, normalisedExtentWithMetadata } from './util/array';
-export { isEqual as isNumberEqual } from './util/number';
-export { tickFormat } from './util/ticks';
-
-export * from './util/time';
-export { createTicks as ticks, tickStep, range, isDenseInterval } from './util/ticks';
+// Only these methods are used by ag-grid.
+// DO NOT ADD EXPORTS UNLESS REQUIRED BY INTEGRATED CHARTS.
```

Comment:
Would it be worth renaming this file to `integrated-utils.ts` or something?

Reply:
Agreed - I believe from offline discussion that is the intent, and this is the first drop of several, so this should happen in a follow-on PR.